### ISSUE TITLE
Errors in mapping cause morph-kgc to produce an exception

### ISSUE THREAD

**Issue by dgarijo** [oeg-upm/morph-kgc#63]
####  What Happens?
Errors in mappings produce an exception. Instead, I was expecting the tool to produce a concrete, explainable error and possibly a line rather than having to read a stack trace).

#### To Reproduce
Use a mapping with errors

**Comment by ArenasGuerreroJulian** [comment]
Since the mapping as an error it is expected to fail. There are already some human friendly feedback for some wrong mappings (e.g, if a Literal termtype is provided in the subject...), and will try to add more to the list.

**Comment by dgarijo** [comment]
I think there are 2 things:

1) Providing the friendly message
2) Ensuring the program does not crash. This issue is more about this one.

**Comment by dgarijo** [comment]
yes, I suggest not to show a stack trace. It looks like the program failed processing something somehow.
